src/pages/AccountRequest/accountUtil.tsx

```diff
@@ -0,0 +1,55 @@
+import { IFormOption } from "components/Form/interfaces";
+
+export interface IUserFormValues {
```

Comment:
I suggest renaming the interface to a more suitable name such as `IAccountRequestForm`.